src/Utilities/mockoptimizer.jl

```diff
@@ -300,6 +300,11 @@ function mock_primal!(mock::MockOptimizer, conduals::Pair...)
     mock.hasprimal = false
     mock_dual!(mock, MOI.InfeasibilityCertificate, conduals...)
 end
+function mock_primal!(mock::MockOptimizer, primstatus::MOI.ResultStatusCode, conduals::Pair...)
```

Comment:
Where is this method called from?

Reply:
From test/contconic.jl lines 26, 32 and 58 (see the changes of this commit)

Reply:
It's not obvious to track how the call to `mock_optimize` gets routed here. Do there really need to 4 versions of `mock_primal!` with overlapping signatures?

Reply:
Is it better with c00a217 ?